tests/src/rules/exports-last.js

```diff
@@ -0,0 +1,117 @@
+import { test } from '../utils'
+
+import { RuleTester } from 'eslint'
+import rule from 'rules/exports-last'
+
+const ruleTester = new RuleTester()
+
+const errors = ['Export statements should appear at the end of the file']
+
+ruleTester.run('exports-last', rule, {
+  valid: [
+    // Empty file
+    test({
+      code: '',
```

Comment:
Ok, try replacing it with a comment
```js
test({
        code: '// comment',
})
```
and if that doesn't work out, just remove the test case.